### PR TITLE
[Issue 403] Update MAINTAINERS.md to Include Core Team

### LINKED ISSUE #403: [Task]: Update "Core Team" with open source runbook content
### Summary

Summary
The USDR team has provided [content in a runbook format ](https://docs.google.com/document/d/1CyoA57KejbNEGhaLoLYlVTYkGER5h4RlSWzBE6xgd3g/edit#heading=h.ap3hp158d2we)to support the Open Source factor of our project. This body of work focuses on incorporating that content into our Grants Equity repo in sections such as, [README.md](https://github.com/HHS/grants-equity/blob/main/README.md), [CONTRIBUTING.md](https://github.com/HHS/grants-equity/blob/main/CONTRIBUTING.md).

This ticket is updating the Grants Equity repo README with the  [Core Team](https://docs.google.com/document/d/1CyoA57KejbNEGhaLoLYlVTYkGER5h4RlSWzBE6xgd3g/edit#bookmark=id.h8u4lot1gkaa) section from the OS runbook.

 

### Acceptance criteria

- [ ] Review the content in the [Core Team](https://docs.google.com/document/d/1CyoA57KejbNEGhaLoLYlVTYkGER5h4RlSWzBE6xgd3g/edit#bookmark=id.h8u4lot1gkaa) and update the content to make relevant for the Grants Equity project where needed
- [ ] Add content in the [Grants Equity README](https://github.com/HHS/grants-equity/blob/main/README.md) section

### PR DESCRIPTION
**Pull request by acouch**
## Summary
Fixes #403

## Changes proposed
Adds core team to the project. Updating the [README.md](./README.md) will be handled in #402 

## Context for reviewers
This is a priority for the USDR work.

### CODE REVIEW

File: MAINTAINERS.md
Comment (SammySteiner) [inline MAINTAINERS.md:9]: Let's include folks' github usernames, so we match those to the correct person on the Maintainers list.

File: MAINTAINERS.md
Comment (SammySteiner) [inline MAINTAINERS.md:3]: Do we want a least one engineer to approve tech changes, at least one designer to approve design changes, etc.?Or keep it as anyone can approve any change?

File: MAINTAINERS.md
Comment (SammySteiner) [inline MAINTAINERS.md:7]: Should we add Julius too?

File: MAINTAINERS.md
Comment (SammySteiner) [inline MAINTAINERS.md:5]: Seems like you're alphabetizing by last name, do you want people to add new folks to the bottom of the list or alphabetized? Maybe mention here in an inline comment for the future?

**Review by SammySteiner — APPROVED** [review]
I left a few suggestions and discussion items.

File: MAINTAINERS.md
Comment (acouch) [reply MAINTAINERS.md:3]: This is a good question. I was thinking that this doc could stay high-level, so that we could make agreements on the team for reviewing types of PRs and to broadly trust the group. For example ADRs reviewers are written on the ticket.

We could go into more depth here and outline specific PR types and who should review them. This would mean we would be obligated to update this if we change a workflow. 

We could update this to sat that **another** member instead of **any**:

> Code for this project is reviewed by another other member of the team before it is merged into the project. 

Or be more prescriptive:

> Code for this project is reviewed by another other member of the team before it is merged into the project. The required reviewers depend on the type of work. For a pull request that is a code change, at least one engineer reviews a code PR. The reviewers for ADRs are listed in the ticket for that ADR. 

Thoughts on either approach?

Comment (andycochran) [reply MAINTAINERS.md:3]: There's also [CODEOWNERS](https://github.blog/2017-07-06-introducing-code-owners/) we could leverage. I'd love to be auto added as a reviewer if code touches CSS.

Comment (andycochran) [reply MAINTAINERS.md:3]: Also wondering if we need to split this into disciplines. I don't wanna appear in both eng and design. But it might be nice for newcomers to know that someone's contributions aren't limited to one discipline. We might consider just listing everyone and their roles in 1 list. 

- [Firstname] [Lastname], [github username], [role]

Maybe there are other things we might include — such as affiliation, timezone, notes about relevant skills/interests, etc.

File: MAINTAINERS.md
Comment (acouch) [reply MAINTAINERS.md:8]: He isn't maintaining the project from a code or design perspective, creating assets, or working in the repository, so my inclination is to leave him off. We can discuss.

File: MAINTAINERS.md
Comment (acouch) [reply MAINTAINERS.md:13]: I based this off of https://github.com/usds/justice40-tool/blob/main/MAINTAINERS.md format, which I think does a nice job of keeping thing simple, while linking to the code owners file.

**Comment by acouch** [comment]
We discussed as a group adding the following folks to CODEOWNERS and keeping the existing format for the maintainers.

**Pull request merged**
